### ISSUE TITLE
NullPointerException: it must not be null

### ISSUE THREAD

**Issue by clankill3r** [hamoid/video_export_processing#72]
Whenever I have a call to `videoExport.endMovie();` I get:

`NullPointerException: it must not be null`

Windows, with processing 4.2.

**Comment by hamoid** [comment]
Hi! :) Did you see the top of the README? Did you try the KotlinGradle branch?

**Comment by clankill3r** [comment]
yeah sorry, it happens with that one, but I posted in the wrong repository by accident.

**Comment by hamoid** [comment]
Ah there's only one repo, you posted on the only one XD. So... I would be happy to receive pull requests to fix any issues. And to pass the repo to a new maintainer as well.

**Comment by vess-dev** [comment]
Hi, still encountering this issue on the KotlinGradle branch.

Minimum viable example reproducing the issue:

```java
import com.hamoid.*;

VideoExport video_handle;

void settings() {
  size(600, 600);
}

void setup() {
  surface.setTitle("test");
  video_handle = new VideoExport(this, "test.mp4");
  video_handle.startMovie();
}

void draw() {
  video_handle.saveFrame();
  video_handle.endMovie();
  exit();
}

**Comment by ajavamind** [comment]
For @vess-dev , I fixed this problem in this fork:
https://github.com/ajavamind/video_export_processing/tree/kotlinGradle
You can build the library on Windows with:
.\gradlew.bat shadowJar

**Comment by vess-dev** [comment]
> For @vess-dev , I fixed this problem in this fork: https://github.com/ajavamind/video_export_processing/tree/kotlinGradle You can build the library on Windows with: .\gradlew.bat shadowJar

Thank you friend. I wish your day warm, and your pillow cold. Much appreciated.
